index.html

```diff
@@ -43,6 +43,7 @@ <h2 class="hero-subtitle">Simple yet flexible JavaScript charting for designers
 				<div class="hero-button-wrapper fade-in animation-delay__6">
 					<a class="button button__red" href="./docs">Get Started</a>
 					<a class="button button__blue" href="./samples">Samples</a>
+					<a class="button button__black" href="https://github.com/chartjs/awesome" target="_blank">Ecosystem</a>
```

Comment:
If you feel to push a new commit, I would pick another color for this button (e.g. `#4BC0C0`).

![image](https://user-images.githubusercontent.com/3874900/104436358-cba0e100-558d-11eb-87bf-f1b2367ddc6e.png)

Reply:
Good idea, I used that colour.